specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/managedCassandra.json

```diff
@@ -0,0 +1,1109 @@
+              },
+              "tokens": {
+                "type": "array",
+                "description": "List of tokens.",
+                "items": {
+                  "type": "string",
+                  "description": "Token of this node."
+                }
+              },
+              "owns": {
+                "type": "number",
+                "format": "double",
+                "description": "The percentage of the data owned by the node per datacenter times the replication factor (e.g., 33.3, or null if the data is not available). For example, a node can own 33% of the ring, but shows 100% if the replication factor is 3. For non-system keyspaces, the endpoint percentage ownership information is shown."
+              },
+              "hostId": {
+                "type": "string",
+                "description": "The network ID of the node."
+              },
+              "rack": {
+                "type": "string",
+                "description": "The rack this node is part of."
+              }
+            }
+          }
+        }
+      }
+    },
+    "BackupResource": {
+      "description": "A restorable backup of a Cassandra cluster.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "cosmos-db.json#/definitions/ARMProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "properties": {
+            "timestamp": {
+              "type": "string",
+              "format": "date-time",
+              "description": "The time this backup was taken, formatted like 2021-01-21T17:35:21"
+            }
+          }
+        }
+      }
+    },
+    "ListBackups": {
+      "description": "List of restorable backups for a Cassandra cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of backups.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/BackupResource"
+          }
+        }
+      }
+    },
+    "ListDataCenters": {
+      "description": "List of managed Cassandra data centers and their properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of data centers.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/DataCenterResource"
+          }
+        }
+      }
+    },
+    "SeedNode": {
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "IP address of this seed node.",
+          "type": "string"
+        }
+      }
+    },
+    "Certificate": {
+      "type": "object",
+      "properties": {
+        "pem": {
+          "description": "PEM formatted public key.",
+          "type": "string"
+        }
+      }
+    },
+    "DataCenterResource": {
```

Comment:
>"DataCenterResource [](http://example.com/codeflow?start=4&length=19)

Does the api support systemData yet? There are a bunch of lint errors about that.

Reply:
No they dont, I have a suppression approval for those.